### DISCUSSION TITLE
Install Kubelogin

### DISCUSSION THREAD

**Discussion by charliewei0716** [actions/virtual-environments#9723] [Ideas]
As discussed in this thread #5598 , for the latest version of Azure AKS, kubelogin is an essential software installation. Please consider this proposal, thank you.

**Comment by Alexey-Ayupov** [comment]
Hello @charliewei0716. It was decided not to add `kubelogin` to the images due to maintenance concerns and relatively short installation time. Please continue installing it in runtime.

**Reply by charliewei0716** [reply]
Hello @Alexey-Ayupov ,

Thank you for your response and for understanding your considerations.

However, I believe that an important use case for the runner-image is in a private network setting, where the duration of the installation should not be a reason for it to be excluded.
